dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/basecomponents/NumberBaseComponent.kt

```diff
@@ -0,0 +1,49 @@
+package org.dataland.frameworktoolbox.intermediate.components.basecomponents
+
+import org.apache.commons.text.StringEscapeUtils
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.intermediate.components.ComponentBase
+import org.dataland.frameworktoolbox.intermediate.components.addStandardCellWithValueGetterFactory
+import org.dataland.frameworktoolbox.specific.datamodel.TypeReference
+import org.dataland.frameworktoolbox.specific.datamodel.elements.DataClassBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+
+/**
+ * The NumberBaseComponent serves as base-class for any number component (percentage, decimal, integer...) and
+ * offers some standard functionality.
+ */
+open class NumberBaseComponent(
+    identifier: String,
+    parent: FieldNodeParent,
+) : ComponentBase(identifier, parent) {
+    open var constantUnitSuffix: String? = null
```

Comment:
Why did this ever needed to be open?

Reply:
I can only imagine Marc doing this for good-practice.
In our case here, we don't need it as open var because none of the inheriting classes will override this anyways.

So should I make it a normal "var" ?

Reply:
It has no use right now. But it doesn't bother me. So do whatever you want. Resolved.